updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -24,3 +24,22 @@ Verify that no RHEL7 workers are present in the 4.10 cluster before performing a
 * All the clusters might update using EUS channels for a conventional update without pools paused, but only clusters with non control-plane `MachineConfigPools` objects can do EUS-to-EUS update with pools paused.
 
 include::modules/updating-eus-to-eus-upgrade.adoc[leveloffset=+1]
+
+include::modules/updating-eus-to-eus-upgrade-console.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="additional-resources_updating-eus-to-eus-upgrade-console"]
+== Additional resources
+
+* xref:../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Preparing for an Operator update]
+* xref:../updating/updating-cluster-within-minor.adoc#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console]
+* xref:../operators/admin/olm-upgrading-operators.adoc[Updating installed Operators]
+
+
+include::modules/updating-eus-to-eus-upgrade-cli.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="additional-resources_updating-eus-to-eus-upgrade-cli"]
+== Additional resources
+
+* xref:../operators/admin/olm-upgrading-operators.adoc[Updating installed Operators]
```

Comment:
All `xref`s to `.adoc` files need a `#` anchor (because Customer Portal requirements), even if it's to the top of the page. In this case, the top of the `olm-upgrading-operators.adoc` has an `[id="olm-upgrading-operators"]` so you can use:

```
* xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Updating installed Operators]
```